BookEcommerce.Models/DAL/Interfaces/IProductVariantRepository.cs

```diff
@@ -9,5 +9,7 @@ namespace BookEcommerce.Models.DAL.Interfaces
 {
     public interface IProductVariantRepository : IRepository<ProductVariant>
     {
+        Task<List<ProductVariant>> GetProductVariantByIdProduct(Guid idProduct);
```

Comment:
GetAllProductVariantsByIdProduct(Guid productId)

Reply:
I will handle it